moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -387,9 +387,10 @@ def trajectory_execution(
             controller_pattern = re.compile("^(.*)_controllers.yaml$")
             possible_names = get_pattern_matches(config_folder, controller_pattern)
             if not possible_names:
-                raise RuntimeError(
-                    "trajectory_execution: `Parameter file_path is undefined "
-                    f"and no matches for {config_folder}/*_controllers.yaml"
+                # Warn the user instead of raising exception
+                logging.warning(
+                    "\x1b[33;20mtrajectory_execution: `Parameter file_path is undefined "
+                    f"and no matches for {config_folder}/*_controllers.yaml\x1b[0m"
```

Comment:
```suggestion
                    "and no matches for {config_folder}/*_controllers.yaml\x1b[0m"
```

Reply:
I'm pretty sure this's needed, why removing it? Printing the value of `config_folder` should make the warning more helpful.

Reply:
is the typo `f` needed?  (Unless that's some crazy Python syntax I'm not aware of)